src/frontend/komponenter/Fagsak/Vedtak/Vedtaksmeny.tsx

```diff
@@ -70,6 +70,12 @@ const Vedtaksmeny: React.FunctionComponent<IVedtakmenyProps> = ({
                     {åpenBehandling.endringstidspunkt && (
                         <EndreEndringstidspunkt åpenBehandling={åpenBehandling} />
                     )}
+                    {toggles[ToggleNavn.trekkILøpendeUtbetaling].valueOf() && (
+                        <Dropdown.Menu.List.Item>
+                            <Calculator />
+                            Legg til trekk i løpende utbetaling
+                        </Dropdown.Menu.List.Item>
```

Comment:
Her mangler det en onClick for å vise/skjule "Trekk i løpende utbetaling" i vedtak-steget. Nå vises dette med en gang man kommer til steget: 
<img width="800" alt="image" src="https://user-images.githubusercontent.com/46678893/204496131-66030eb7-980f-4feb-ba66-c1cdb61e02d7.png">

Reply:
Kan f.eks. ha en boolean som settes til true on click som sendes med til komponenten for trekk i løpende betaling og bestemmer om den skal vises eller ikke. Litt usikker på hvordan dette fungerer med lagring til senere